docs/nGQL-tutorial.md

```diff
@@ -311,9 +307,10 @@ you to follow these syntax norms:
   - eg: inService
 
 
-  | Graph entity  | Recommended style | Example |
-  |:-: | :-: | :-: |:-: |
-  |Key words | Upper case   | SHOW SPACES     |
-  |Vertex tags | Upper camel case, beginning with an upper-case character   | ManageTeam   |
-  |Edges | Upper snake case, beginning with an upper-case character   | Play_for   |
-  |Property names | Lower camel case, beginning with a lower-case character   | inService   | --> -->
+| Graph entity  | Recommended style                                          | Example     |
+|:-:            | :-:                                                        | :-:         |
+|Key words      | Upper case                                                 | SHOW SPACES |
+|Vertex tags    | Upper camel case, beginning with an upper-case character   | ManageTeam  |
```

Comment:
It looks like the unittest, and our examples are different.

How about we comment out the style section? until we have agreements.

Reply:
Correctly! We should keep the style unify.